composer/datasets/ade20k.py

```diff
@@ -20,9 +26,11 @@
 from composer.datasets.utils import NormalizationFn, pil_image_collate
 from composer.utils import dist
 
+__all__ = ["ADE20k", "ADE20kDatasetHparams", "ADE20kWebDatasetHparams"]
+
 
 class RandomResizePair(torch.nn.Module):
-    """Resize the image and target to `base_size` scaled by a randomly sampled value.
+    """Resize the image and target to ``base_size`` scaled by a randomly sampled value.
 
     Args:
         min_scale (float): the minimum value the samples can be rescaled.
```

Comment:
base_size (and many parameters in this file) do not have a default specified or need the default formatted correctly. Defaults should be specified with the following formatting:
```
Default: ``value``.
```